src/robots/yamcs_TMTC.py

```diff
@@ -60,15 +64,73 @@ def _command_callback(self, command:CommandHistory):
+            self.handle_high_res_capture()
         elif name == self._yamcs_conf["commands"]["camera_streaming_on_off"]:
             self._set_activity_of_camera_streaming(arguments["action"])
         elif name == self._yamcs_conf["commands"]["camera_capture_depth"]:
-            self._camera_handler.transmit_camera_view(CameraViewTransmitHandler.BUCKET_IMAGES_DEPTH, "high", "depth")
+            self.handle_depth_capture()
+        elif name == self._yamcs_conf["commands"]["power_electronics"]:
+            self._handle_electronics(arguments["subsystem_id"], arguments["power_state"])
+        elif name == self._yamcs_conf["commands"]["solar_panel"]:
+            self._handle_solar_panel(arguments["deployment"])
+        elif name == self._yamcs_conf["commands"]["go_nogo"]:
+            self._handle_go_nogo(arguments["decision"])
         # here add reactions to other commands
         else:
             print("Unknown command:", name)
 
+    def handle_high_res_capture(self):
+        self._camera_handler.transmit_camera_view(CameraViewTransmitHandler.BUCKET_IMAGES_ONCOMMAND, "high", "rgb")
+        self._set_obc_state(ObcState.CAMERA, 10)
+
+    def handle_depth_capture(self):
+        self._camera_handler.transmit_camera_view(CameraViewTransmitHandler.BUCKET_IMAGES_DEPTH, "high", "depth")
+        self._set_obc_state(ObcState.CAMERA, 10)
+
+    # def _set_obc_to_idle_after_time(self, time=10):
+    #     if self._intervals_handler.does_exist(IntervalName.OBC_STATE.value):
+    #         self._intervals_handler.update_next_time(IntervalName.OBC_STATE.value, time)
+    #     else:
+    #         self._intervals_handler.add_new_interval(name=IntervalName.STOP_ROBOT.value, seconds=time, is_repeating=False, execute_immediately=False,
+    #                                              function=self.self._robot.subsystems.set_obc, f_args=[ObcState.IDLE])
+
+    def _set_obc_state(self, state:ObcState, set_to_idle_after=0):
+        if self._intervals_handler.does_exist(IntervalName.OBC_STATE.value):
+            self._intervals_handler.remove_interval(IntervalName.OBC_STATE.value)
+
+        self._robot.subsystems.set_obc(state)
+
+        if set_to_idle_after != 0:
+            self._intervals_handler.add_new_interval(name=IntervalName.OBC_STATE.value, seconds=set_to_idle_after, is_repeating=False, execute_immediately=False,
+                                                 function=self._robot.subsystems.set_obc, f_args=[ObcState.IDLE])
+
+    def _handle_solar_panel(self, new_state:SolarPanelState):
+        if new_state == SolarPanelState.STOWED:
+            self._robot.subsystems.stow_solar()
+        elif new_state == SolarPanelState.DEPLOYED:
+            self._robot.subsystems.deploy_solar()
+        else:
+            print("New state for solar panel is uknown:", new_state)
+
+    def _handle_electronics(self, electronics:str, new_state:PowerState):
```

Comment:
the logic of both handle electornics and handle gonogo should be the same? 
maybe make consitent with the cleaner set_*_state() as in the gonogo implementation?